### PR TITLE
Added deployment instructions to the readme

### PR DESCRIPTION
**Pull request by tbusath8**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,18 @@ linc-cv uses 3 components: [Flower](https://flower.readthedocs.io/en/latest/), [
...
+    
+## Deployment
+This application is currently deployed via a blue green methodology using Github Actions. The process is as follows:
+1. Work on code changes in a feature branch based off of the staging branch
+2. Submit a merge request to the staging branch
+3. Run the ***Deploy*** Github Action workflow pointed to the staging branch
+4. Receive approval from project administrators
+5. Submit a merge request to the master branch
+6. Determine which environment (blue, green) is active in production
+7. Run the ***Deploy*** Github Action workflow pointed to the inactive branch
```
Comment (shihgianlee) [inline README.md:142]: What does inactive branch mean here?

**Comment by shihgianlee** [comment]
Please move the deployment section up, before the `Local setup for Mac` section.

File: README.md [hunk 2]
```diff
@@ -130,6 +146,18 @@ linc-cv uses 3 components: [Flower](https://flower.readthedocs.io/en/latest/), [
...
+8. Point the webapp in Heroku to the inactive branch, making it active
```
Comment (shihgianlee) [inline README.md:159]: We should specify pointing both staging and production webapp to the new AWS instance.

File: README.md (hunk 1 above)
Comment (tbusath8) [reply README.md:158]: I should have said the inactive environment

**Pull request merged**